resources/frontend_client/index_template.html

```diff
@@ -56,42 +67,39 @@
   <body>
     <div id="root"></div>
 
+    <!-- Using the Web Font Loader lets us load the fonts asynchronously for faster page loads -- see https://github.com/typekit/webfontloader -->
+    <!-- If you modify this script, make sure you update the whitelisted Content-Security-Policy hash in metabase.middleware.security -->
     <script type="text/javascript">
-      // Load scripts asyncronously after the page has finished loading
-      (function () {
-        function loadScript(src, onload) {
-          var script = document.createElement('script');
-          script.type  = "text/javascript";
-          script.async = true;
-          script.src   = src;
-          if (onload) script.onload = onload;
-          document.body.appendChild(script);
-        }
-        loadScript('https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js', function () {
-          WebFont.load({ google: {
-            families: ["Lato:300,400,700,900"] }
-          });
-        });
-        var googleAuthClientID = window.MetabaseBootstrap.google_auth_client_id;
-        if (googleAuthClientID) {
-          loadScript('https://apis.google.com/js/api:client.js');
+      WebFontConfig = {
+        google: {
+          families: ["Lato:300,400,700,900"]
         }
-      })();
+      };
     </script>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" async></script>
 
-    <script type="text/javascript">
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-      // if we are not doing tracking then go ahead and disable GA now so we never even track the initial pageview
-      const tracking = window.MetabaseBootstrap.anon_tracking_enabled;
-      const ga_code = window.MetabaseBootstrap.ga_code;
-      if (!tracking) {
-        window['ga-disable-'+ga_code] = true;
-      }
-
-      ga('create', ga_code, 'auto');
-    </script>
+    <!-- If Google Auth is enabled load the Google Auth JS lib -->
+    {{#enableGoogleAuth}}
+      <script type="text/javascript" src="https://apis.google.com/js/api:client.js" async></script>
+    {{/enableGoogleAuth}}
+
+    <!-- Google Analytics -->
+    <!-- If you modify this script, make sure you update the whitelisted Content-Security-Policy hash in metabase.middleware.security -->
+    {{#enableAnonTracking}}
```

Comment:
added bonus: if anonymous tracking is disabled, don't waste time loading Google Analytics